pom.xml

```diff
@@ -184,7 +185,19 @@
           <xmlOutput>true</xmlOutput>
           <effort>Max</effort>
           <threshold>Low</threshold>
-          <excludeFilterFile>${findbugs.config}</excludeFilterFile>
+          <excludeFilterFile>${project.build.directory}/plugin-config/findbugs/hotels-findbugs-exclude.xml</excludeFilterFile>
```

Comment:
Hmm, this is a bit weird, what we should have is a property defined at the top along with the other properties called `findbugs.config` which by default is set to this value so it can be overridden downstream if teams want to use the plugin but with a different config file. See our internal parent for how we do it. Then in addition we should have a `spotbugs.config` pointing to a different `hotels-spotbugs-exclude.xml` (which you're going to need to add to `hotels-oss-plugin-config` and refer to from here and then update the spotbugs plugin to use that property for the config file.

Reply:
I've added in the properties, is it the right path to be using for this project?

Reply:
I think one can do this slightly differently, see my comments.